grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2813,4 +2815,34 @@ class StockMovementService {
         }
         return true
     }
+
+    void rollbackApproval(String stockMovementId) {
+        StockMovement stockMovement = getStockMovement(stockMovementId)
+        // TODO: remove .get() from current user in Grails 3 (with .get it will return null)
+        if (!canRollbackApproval(AuthService.currentUser.get(), stockMovement)) {
+            String errorMessage = applicationTagLib.message(
+                    code: "request.errors.approval.rollback.message",
```

Comment:
Now that I've seen this, I'm convince the messages should follow the convention
```
request.rollbackApproval...
```
and this particular message should be 
```
request.rollbackApproval.insufficientPermissions.message
```